entity-framework/toc.md

```diff
@@ -127,12 +127,142 @@
+### [Framework Runtime Version for EF Designer Models](selecting-entity-framework-runtime-version-for-ef-designer-models-ef6-onwards.md)
+### [Logging and Intercepting Database Operations](entity-framework-logging-and-intercepting-database-operations-ef6-onwards.md)
+### [Upgrading to Entity Framework 6](upgrading-to-entity-framework-6.md)
+### [Model-Based Conventions](entity-framework-model-based-conventions-ef6-onwards.md)
+### [IDbDependencyResolver Services](entity-framework-idbdependencyresolver-services-ef6-onwards.md)
+### [Pre-Generated Mapping Views](entity-framework-pre-generated-mapping-views.md)
+### [Handling of Transaction Commit Failures](entity-framework-handling-of-transaction-commit-failures-ef6-1-onwards.md)
+### [Tools Keyboard Shortcuts](entity-framework-tools-keyboard-shortcuts.md)
+### [Validation](entity-framework-validation.md)
+### [Testability and Entity Framework 4.0](testability-and-entity-framework-4-0.md)
+### [Entity Framework Tools](ado-net-entity-framework-tools.md)
+### [Get Help Using Entity Framework](get-help-using-entity-framework.md)
+### Entity Framework Licenses
+#### [EF5 License (ENU)](entity-framework-5-license.md)
+#### [EF5 License (CHS)](entity-framework-5-license-chs.md)
+#### [EF5 License (CHT)](entity-framework-5-license-cht.md)
+#### [EF5 License (FRA)](entity-framework-5-license-fra.md)
+#### [EF5 License (DEU)](entity-framework-5-license-deu.md)
+#### [EF5 License (ITA)](entity-framework-5-license-ita.md)
+#### [EF5 License (JPN)](entity-framework-5-license-jpn.md)
+#### [EF5 License (KOR)](entity-framework-5-license-kor.md)
+#### [EF5 License (RUS)](entity-framework-5-license-rus.md)
+#### [EF5 License (ESN)](entity-framework-5-license-esn.md)
+#### [EF6 Runtime License (ENU)](entity-framework-6-runtime-license.md)
+#### [EF6 Runtime License (CHS)](entity-framework-6-runtime-license-chs.md)
+#### [EF6 Runtime License (CHT)](entity-framework-6-runtime-license-cht.md)
+#### [EF6 Runtime License (FRA)](entity-framework-6-runtime-license-fra.md)
+#### [EF6 Runtime License (DEU)](entity-framework-6-runtime-license-deu.md)
+#### [EF6 Runtime License (ITA)](entity-framework-6-runtime-license-ita.md)
+#### [EF6 Runtime License (JPN)](entity-framework-6-runtime-license-jpn.md)
+#### [EF6 Runtime License (KOR)](entity-framework-6-runtime-license-kor.md)
+#### [EF6 Runtime License (RUS)](entity-framework-6-runtime-license-rus.md)
+#### [EF6 Runtime License (ESN)](entity-framework-6-runtime-license-esn.md)
+#### [EF6 Runtime Alpha License](ef6-runtime-alpha-license.md)
+#### [EF6 Runtime Beta/RC License](ef6-runtime-beta-rc-license.md)
 
-### [⤤ API Reference](https://docs.microsoft.com/dotnet/api/?view=efcore-2.0)
 
-## [Entity Framework 6](ef6/index.md)
-### [⤤ Documentation](http://msdn.com/data/ef)
 ### [⤤ API Reference](https://msdn.microsoft.com/library/dn223258.aspx)
```

Comment:
this should be migrated as well. are you working on that with someone?

Reply:
Not yet. Any suggestion on who to talk to?

Reply:
@dend can you help? we'd need to migrate EF6 API reference to the .NET API Browser.